src/olympia/files/forms.py

```diff
@@ -76,10 +98,10 @@ def option(files, label=None, deleted=False, channel=None):
                         option(f, deleted=ver.deleted, channel=channel))
                 output.append(u'</optgroup>')
 
-        return jinja2.Markup(u''.join(output))
+        return output
```

Comment:
This can't be easily uplifted since the APIs are quite different between 1.11 and 1.8. I'd keep it here tbh